test/new-e2e/utils/e2e/client/agent.go

```diff
@@ -30,6 +35,56 @@ func (agent *Agent) init(auth *Authentification, data *agent.ClientData) error {
 	return err
 }
 
-func (agent *Agent) Status() (string, error) {
-	return agent.sshClient.Execute("sudo datadog-agent status")
+func (agent *Agent) Version() (string, error) {
+	return agent.sshClient.Execute("datadog-agent version")
+}
+
+type Status struct {
+	rawString string
+}
+
+func NewStatus(s string) *Status {
```

Comment:
I think `Status` shouldn't have a public constructor as it should only be created by the `Status` command. What do you think?